test/coveralls.test.js

```diff
@@ -3,19 +3,80 @@
 const Code = require('code')
 const Lab = require('lab')
 const Seneca = require('seneca')
+var _ = require('lodash')
+
+const Proxyquire = require('proxyquire')
+const NpmSenecaFakeData = require('./npm-data')
+const CoverallsSenecaFakeData = require('./coveralls-data')
+const InvalidPluginName = 'qwerty_qwerty'
 
 const lab = exports.lab = Lab.script()
 const describe = lab.describe
-const suite = lab.suite
 const before = lab.before
 const it = lab.it
 const expect = Code.expect
 
+const NpmRegistry = 'http://registry.npmjs.org/'
+
+const RequestMap = [
+  {
+    urlMatch: NpmRegistry + 'seneca',
+    err: null,
+    response: {},
+    body: JSON.stringify(NpmSenecaFakeData)
+  },
+  {
+    urlMatch: InvalidPluginName,
+    err: null,
+    response: {},
+    body: '{}'
+  },
+  {
+    urlMatch: 'coveralls.io',
+    err: null,
+    response: {},
+    body: JSON.stringify(CoverallsSenecaFakeData)
+  }
+]
+
+// const DefaultRequestMap = [
+//   {
+//     urlMatch: NpmRegistry + 'seneca',
+//     err: null,
+//     response: {},
+//     body: JSON.stringify(NpmSenecaFakeData)
+//   },
+//   {
+//     urlMatch: 'coveralls.io',
+//     err: null,
+//     response: {},
+//     body: JSON.stringify(CoverallsSenecaFakeData)
+//   }
+// ]
+
+const RequestProxy = {
+  request: {
+    get: (opts, done) => {
+      const request = _.find(RequestMap, (o) => {
```

Comment:
you can write this in a simple way, because you have only one arg, there is no need to put the () and because the function block contains only the return statement you can remove the {}, so this will be:
`o => opts.url.includes(o.urlMatch)`